pkgs/os-specific/linux/minimal-bootstrap/stage0-posix/default.nix

```diff
@@ -3,6 +3,8 @@
 }:
 
 lib.makeScope newScope (self: with self; {
+  inherit (callPackage ./platforms.nix { }) platforms stage0Arch m2libcArch m2libcOS baseAddress;
+
   inherit (self.callPackage ./bootstrap-sources.nix {})
     version hex0-seed minimal-bootstrap-sources;
```

Comment:
eval fails

```suggestion
    version hex0-seed minimal-bootstrap-sources stage0Arch;
```

Reply:
Forgot that `bootstrap-sources.nix` is called from `make-bootstrap-sources.nix`. Since `make-bootstrap-sources.nix` is meant to be platform agnostic I've instead removed the `stage0Arch` argument again and moved `hex0-seed` from `bootstrap-sources.nix` to `hex0.nix`.